services/coderabbit/coderabbit-stats.tester.js

```diff
@@ -0,0 +1,46 @@
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('live CodeRabbitStats')
+  .get('/stats/github/coderabbitai/ast-grep-essentials.json')
+  .expectBadge({
+    label: 'CodeRabbit Reviews',
+    message: /^\d+$/,
+  })
+
+t.create('CodeRabbitStats valid repo')
+  .get('/stats/github/coderabbitai/ast-grep-essentials.json')
+  .intercept(nock =>
+    nock('https://api.coderabbit.ai')
+      .get('/stats/github/coderabbitai/ast-grep-essentials')
+      .reply(200, { reviews: 101 }),
+  )
+  .expectBadge({
+    label: 'CodeRabbit Reviews',
+    message: '101',
+  })
+
+t.create('CodeRabbitStats repo not found')
```

Comment:
Could we use live examples for all types of responses?

Reply:
Made all tests use the live service endpoint.